modules/shared/src/main/scala/org/tessellation/schema/transaction.scala

```diff
@@ -73,11 +73,21 @@ object transaction {
     val ordinal: TransactionOrdinal = parent.ordinal.next
   }
 
+  implicit val transactionOrder: Order[Transaction] = (x: Transaction, y: Transaction) =>
+    implicitly[Order[BigInt]].compare(x.ordinal.coerce, y.ordinal.coerce)
+
+  implicit val transactionOrdering: Ordering[Transaction] = transactionOrder.toOrdering
+
   implicit val signedTransactionOrder: Order[Signed[Transaction]] = (x: Signed[Transaction], y: Signed[Transaction]) =>
-    implicitly[Order[BigInt]].compare(x.value.ordinal.coerce, y.value.ordinal.coerce)
+    implicitly[Order[Transaction]].compare(x.value, y.value)
```

Comment:
hmm, I see there is already a generic order for `Signed[A]`. maybe you could reuse it? 🤔